openfisca_france_dotations_locales/parameters/dotation_solidarite_rurale/perequation/attribution/plafond_ratio_progression.yaml

```
@@ -0,0 +1,5 @@
+description: Plafond de l'attribution de la DSR fraction péréquation pour une commune éligible en fonction de son attribution de l'année précédente
+reference: https://www.legifrance.gouv.fr/affichCodeArticle.do?idArticle=LEGIARTI000036433094&cidTexte=LEGITEXT000006070633
+values:
+  2019-01-01:
```

En, [L2334-22](https://www.legifrance.gouv.fr/affichCode.do;jsessionid=91AB41F882E43E9E9E550ED908348012.tplgfr38s_3?idSectionTA=LEGISCTA000006197650&cidTexte=LEGITEXT000006070633&dateTexte=20200901) il est dit : 
```
A compter de 2012, l'attribution au titre de cette fraction d'une commune éligible ne peut être ni inférieure à 90 % ni supérieure à 120 % du montant perçu l'année précédente.
```

```suggestion
  2012-01-01:
```